src/dialog/overflow-menu/overflow-menu-option.component.ts

```diff
@@ -24,7 +24,6 @@ import {
 			class="bx--overflow-menu-options__btn"
 			role="menuitem"
 			[ngClass]="{
-				'bx--overflow-menu-options__option--danger': type === 'danger',
 				'bx--overflow-menu-options__option--disabled': disabled
```

Comment:
Considering this class is now applied on host, should it be removed from here too?
If yes, this `ngClass` should be removed too.

Reply:
So since you have `tabindex="-1"` and  `disabled="true"` (it's missing now, so that's be good to add) you can use `${buttonClass}:not([disabled])` instead and drop the disabled class from here